syslogdiag/emailing.py

```diff
@@ -94,10 +105,9 @@ def get_email_details():
         email_server = production_config.email_server
         email_to = production_config.email_to
         email_port = production_config.email_port
+        use_gmail = production_config.use_gmail
```

Comment:
This does not return None if use_gmail is missing, instead it will break

See my comment above